lib/floki.ex

```diff
@@ -84,9 +84,9 @@ defmodule Floki do
 
   """
```

Comment:
add options to the docs

Reply:
Can you add an example of the `encode: false` here too?